static/app/views/monitors/components/monitorQuickStartGuide.tsx

```diff
@@ -0,0 +1,94 @@
+import {Fragment, useState} from 'react';
+import styled from '@emotion/styled';
+
+import {CompactSelect} from 'sentry/components/compactSelect';
+import {PlatformKey} from 'sentry/data/platformCategories';
+import {t} from 'sentry/locale';
+import {SelectValue} from 'sentry/types';
+import {
+  CLICronQuickStart,
+  CurlCronQuickStart,
+  PythonCronQuickStart,
+  QuickStartProps,
+} from 'sentry/views/monitors/components/quickStartEntries';
+
+interface Props {
+  monitorSlug?: string;
+  platform?: PlatformKey;
+}
+
+interface OnboardingGuide {
+  guide: (props: QuickStartProps) => React.ReactNode;
+  label: string;
+  alwaysShown?: boolean;
+  platforms?: Set<PlatformKey>;
+}
+
+const onboardingGuides: Record<string, OnboardingGuide> = {
+  cli: {
+    label: 'CLI',
+    guide: ({slug}: QuickStartProps) => <CLICronQuickStart slug={slug} />,
+    alwaysShown: true,
```

Comment:
No need to define some as always shown or not. All options should always be shown for any platform, the project selected only changes the _default selection_.

Otherwise, if someone selects a project type "Other" they will only see CLI and cURL.

Reply:
done